connectors/sources/dropbox.py

```diff
@@ -987,6 +1000,7 @@ def get_email(self, permission, identity):
     async def get_permission(self, permission, account_id):
         permissions = []
         if identities := permission.get("users"):
+            self._logger.debug("Fetching users")
```

Comment:
I'm unclear on what exactly is being logged here. There are no API calls happening to fetch users. Also there are no logs for `invitees` or `groups`. I think this log is unnecessary.